File: db-4.8.30/examples_csharp/ex_txn/PayloadData.cs
/*-
 * See the file LICENSE for redistribution information.
 *
 * Copyright (c) 2009 Oracle.  All rights reserved.
 *
 */
using System;
using System.Collections.Generic;
using System.Text;

namespace ex_txn {
    [Serializable()]
    public class PayloadData {
        private int oID;
        private string threadName;
        private double doubleData;

        public PayloadData(int id, string name, double data) {
            oID = id;
            threadName = name;
            doubleData = data;
        }

        public double DoubleData {
            get { return doubleData; }
            set { doubleData = value; } 
        }

        public int ID {
            get { return oID;  }
            set { oID = value;  }
        }

        public string ThreadName {
            get { return threadName;  }
            set { threadName = value;  }
        }
    }
}

File: db-4.8.30/examples_java/src/db/txn/PayloadData.java
/*-
 * See the file LICENSE for redistribution information.
 *
 * Copyright (c) 2005-2009 Oracle.  All rights reserved.
 *
 * $Id$ 
 */

package db.txn;

import java.io.Serializable;

public class PayloadData implements Serializable {
    private int oID;
    private String threadName;
    private double doubleData;

    PayloadData(int id, String name, double data) {
        oID = id;
        threadName = name;
        doubleData = data;
    }

    public double getDoubleData() { return doubleData; }
    public int getID() { return oID; }
    public String getThreadName() { return threadName; }
}
